layouts/article.html

```diff
@@ -16,7 +16,8 @@
           <% section = lookup_section('toc', @item[:title]) %>
           <h1>
             <span class="article-title"><%= section << ' ' << @item[:title] if section %></span>
-            <a class="improve-link" href="<%= "https://github.com/atom/flight-manual.atom.io/edit/master/content#{@item.identifier}" %>">
+            <a class="improve-link"
+                href="<%= "https://github.com/atom/flight-manual.atom.io/edit/master/content#{@item.identifier}" %>" data-proofer-ignore>
```

Comment:
@gjtorikian - I think I've taken care of [inlining the link and proof-ignoring][1]. How can I run the test suite locally to make sure I got the `data-proofer-ignore` part right?

[1]: https://github.com/atom/flight-manual.atom.io/pull/181#discussion_r57410712